pkgs/top-level/python-packages.nix

```
@@ -7861,6 +7861,8 @@ self: super: with self; {
 
   pyutil = callPackage ../development/python-modules/pyutil { };
 
+  pyunpack = callPackage ../development/python-modules/pyunpack { };
```

```suggestion
  pyunpack = callPackage ../development/python-modules/pyunpack { };

  pyutil = callPackage ../development/python-modules/pyutil { };
```
Please sort by alphabetical order.